src/main/java/net/pms/formats/Format.java

```diff
@@ -415,6 +415,7 @@ public static boolean isSupportedMimetype(String mimetype) {
 		if ("audio/L24".equalsIgnoreCase(mimetype)
 				|| "video/vnd.dlna.mpeg-tts".equalsIgnoreCase(mimetype)
 				|| "video/mpeg2".equalsIgnoreCase(mimetype)
+				|| "audio/vnd.dlna.adts".equals(mimetype)
```

Comment:
@Sami32 ```mimetype.equals("audio/vld.dlna.adts")``` and ```"audio.vnd.dlna.adts".equals(mimetype)``` does the same thing but the last form is "NPE safe" (because the fixed string will never be ```null```).

I'm not sure if that was what you were wondering though. You have to use ```equals()``` when comparing strings and other classes in Java, using ```=``` would only compare the memory address in which the class resides (NOT what you want).